dcos/jsonitem.py

```diff
@@ -283,8 +283,9 @@ def _parse_url(value):
     if value_regex is None:
         scheme_match = re.match(scheme_pattern, value, re.IGNORECASE)
         if scheme_match is None:
-            msg = 'Please check url {!r}. Missing http(s)://'.format(value)
-            raise DCOSException(msg)
+            logger.debug("Defaulting URL to https scheme")
+            value = "https://" + value.lstrip(scheme_pattern)
```

Comment:
I was trying to be have scheme_pattern "clean up" and invalid scheme. For example, value = "httpdcos.com" would be corrected to "https://dcos.com"

But you're right I should just add the scheme to the value as is.